bin/spice/pkg/github/runtime_release.go

```diff
@@ -114,9 +126,18 @@ func get_ai_accelerator() (string, bool) {
 		if err != nil {
 			slog.Error("checking for CUDA device", "error", err)
 		}
-		if version != nil {
-			return "cuda_" + *version, true
+
+		if version == nil {
+			return "", false
+		}
+
+		if !checkCudaVersionSupported(*version) {
+			slog.Warn(fmt.Sprintf("The GPU Compute Capability '%s' is not supported by Spice GPU builds. Supported versions are: %v. Falling back to CPU.",
```

Comment:
This is to technical of an error message. Let's make sure to follow the error guidelines. Most people don't know what a compute capability is, let's just say that the GPU isn't supported.

Reply:
Thank you, updated to

>"GPU version is not supported. Spice will run but without GPU acceleration. Detected GPU version: <version>